datatree/datatree.py

```diff
@@ -853,6 +853,22 @@ def __setitem__(
         else:
             raise ValueError("Invalid format for key")
 
+    def move(self, original_path: str, to: str) -> DataTree:
+        new_tree = self.copy()
+        target_node = new_tree.__getitem__(original_path)
+        path_to_parent = "/".join(original_path.split("/")[:-1])
```

Comment:
We have a relative_to method for this - if you look at the bottom of `treenode.py` there is a `NodePath` class you can use that will be much cleaner.

Reply:
Here I'm trying to get the path to parent. I'm really not sure how `relative_to` is going to help me here given that I only have the path to the node itself

Reply:
In the TreeNode Class, there is a method, _set_parent ( ) , it takes, new_parent, and, child_name, as arguments. I wonder if this would simplify our move method. Since moving the node is actually re-assigning parent. @JosiasAurel , @TomNicholas

Reply:
@TomNicholas please do you mind checking my latest changes? Thank you